vuln/core/11.json

```diff
@@ -0,0 +1,9 @@
+{
+  "cve": [],
+  "vulnerable": "6.x",
+  "patched": "^6.9.0",
+  "ref": "https://nodejs.org/en/blog/vulnerability/october-2016-security-releases/ for",
+  "author": "Jann Horn",
```

Comment:
Reasonably, but that should happen first in https://github.com/nodejs/security-wg/pull/34, and then applied to vuln/core as well as vuln/npm